### DISCUSSION TITLE
Open to funding?

### DISCUSSION THREAD

**Discussion by lorenzwalthert** [python-poetry/poetry#6661] [General] [answered]
At work, we extensively use poetry and benefit from it. Great job. We'd like to support the ongoing development financially, but could not figure out a way to do so. Would you consider giving your user this option, e.g. throug GitHub Sponsors or Tidelift?

**Comment by neersighted** [comment, marked as answer]
I answered this the other day in the Poetry Discord -- the answer is not at this time:

> There is currently no way to contribute funds to Poetry development. We have discussed a relationship with the PSF, or membership in the PyPA, as avenues to rapidly create infrastructure for funding Poetry development; however, that work is still very much a long term aspiration (and likewise, independent infrastructure is even more of a pipe dream). [snip unrelated discussion]

**Reply by lorenzwalthert** [reply]
Thanks for clarifying @neersighted.
